### PR TITLE
Add a test of the actual default output of `conduit install`.

### PR DESCRIPTION
**Pull request by briansmith**
This test is useful to make it clear when we change the default
settings of `conduit install`.

Signed-off-by: Brian Smith <brian@briansmith.org>

### CODE REVIEW

**Comment by briansmith** [comment]
This is based on #373. I will rebase it onto master before merging.

File: cli/cmd/testdata/install_default.golden [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: api
+  namespace: conduit
+  labels:
+    conduit.io/control-plane-component: controller
+  annotations:
+    conduit.io/created-by: conduit/cli latest
```
Comment (siggy) [inline cli/cmd/testdata/install_default.golden]: i think the version values need to be `undefined`?

```bash
$ go test ./...
?   	github.com/runconduit/conduit/cli	[no test files]
--- FAIL: TestRender (0.03s)
    --- FAIL: TestRender/1:_testdata/install_default.golden (0.02s)
    	test_helper.go:18: Unexpected output:
    		@@ -1681,38 +1681,41 @@
    		 by: conduit/cli
    		-lat
    		+undefin
    		 e
    		-st
    		+d
    		 %0Aspec:%0A  type: C
```

File: cli/cmd/testdata/install_default.golden [hunk 2]
```diff
@@ -0,0 +1,548 @@
...
+        conduit.io/control-plane-ns: conduit
+    spec:
+      containers:
+      - args:
+        - -addr=:8084
+        - -metrics-addr=:9994
+        - -api-addr=api:8085
+        - -static-dir=/dist
+        - -template-dir=/templates
+        - -uuid=deaab91a-f4ab-448a-b7d1-c832a2fa0a60
```
Comment (siggy) [inline cli/cmd/testdata/install_default.golden]: insert after this line:
```
- -controller-namespace=conduit
```

**Comment by briansmith** [comment]
@siggy I rebased this on top of master. Could you please look at it again and approve it? Thanks!

**Pull request merged**